tests/sqllogictests/suites/base/20+_others/20_0017_describe_view.test

```diff
@@ -0,0 +1,90 @@
+statement ok
+use default
+
+statement ok
+DROP VIEW IF EXISTS v
+
+statement ok
+DROP TABLE IF EXISTS t
+
+statement ok
+CREATE TABLE t(a bigint null, b int null, c varchar(255) null, d smallint not null, e Date not null ) ENGINE = Null
+
+statement ok
+CREATE VIEW v AS SELECT * FROM t
+
+onlyif http
```

Comment:
If use MySQL Handler the result of `desc view <view_name>` will return empty result. I think maybe because MySQL not  support this syntax. So in sqllogic test only need test http handler. cc @sundy-li

Reply:
ok